### ISSUE TITLE
Docker image of version 2.21.3 seems broken

### ISSUE THREAD

**Issue by grignards** [jdkato/txtlint#537]
The docker image (https://hub.docker.com/layers/jdkato/vale/v2.21.3) seems to be broken:

```shell
docker run -ti --rm jdkato/vale:v2.21.3
Unable to find image 'jdkato/vale:v2.21.3' locally
v2.21.3: Pulling from jdkato/vale
Digest: sha256:8c37a7cf819497f029f407abc08ec039839e5b04c67cf48d411f13ac697f3b95
Status: Downloaded newer image for jdkato/vale:v2.21.3
docker: Error response from daemon: failed to create shim: OCI runtime create failed: runc create failed: unable to start container process: exec: "/bin/vale": stat /bin/vale: no such file or directory: unknown.
```

v2.21.2 works fine.

**Comment by jdkato** [comment]
Should be fixed now.
